mobile/src/main/res/values/strings.xml

```diff
@@ -66,19 +66,37 @@
     <string name="info_voice_input">"openHAB, at your command!"</string>
     <string name="info_voice_recognized_text">"Recognized command: %1$s"</string>
     <string name="info_demo_mode">Running in demo mode. To connect to your openHAB server please go to settings menu.</string>
+    <string name="info_demo_mode_short">Running in demo mode</string>
     <string name="info_conn_url">Connecting to local URL</string>
     <string name="info_conn_rem_url">Connecting to remote URL</string>
     <string name="info_discovery">Discovering openHAB. Please wait...</string>
     <string name="error_no_url">Please check that openHAB is running or configure openHAB URL or remote URL</string>
-    <string name="error_authentication_failed">Authentication Failed. Please check Username/Password settings</string>
     <string name="error_empty_sitemap_list">openHAB returned empty sitemap list</string>
     <string name="error_network_not_available">Network is not available</string>
+    <string name="error_http_connection_failed">Got invalid answer from openHAB. Please check if openHAB is working correctly (HTTP response code %d)</string>
```

Comment:
Does it make clear, that oh server responed, but a http error occured?

Reply:
I'm ok with it, but i would say "Received an invalid response from openHAB.  Please check if openHAB is working correctly (HTTP response code %d)"